AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -210,19 +223,19 @@ The Alembic package includes the following Materials:
 |__Points Motion Vectors__      |{TODO} |
 |__Standard__                   |Standard PBR material with motionblur support added |
 |__Standard (Roughness setup)__ |Standard (Roughness setup) PBR with roughness material with motionblur support added |
-|__Standard__(Specular setup)   |Standard (Specular setup) material with motionblur support added |
+|__Standard (Specular setup)__   |Standard (Specular setup) material with motionblur support added |
 
 ## Motion Blur
 
 The Alembic shaders included add motion vector generation. This is useful for rendering that requires motion vectors, such as the post processing effect MotionBlur. If you want to add the motion vector generation function to your own shader, add the line `UsePass "Hidden / Alembic / MotionVectors / MOTIONVECTORS"` into SubShader. Please see AlembicMotionVectors.cginc for details. Since the velocity data is passed to the fourth UV, the apex position of the previous frame is calculated based on it. Left is unprocessed, right is output of motion vector and MotionBlur applied by the Post Processing Stack.<br/>![Alembic MotionBlur](images/abc_motionblur.png)|
 
 <a name="dccs"></a>
-# Working with Autodesk Maya® 
+# Working with Autodesk® Maya® 
 
-Maya's shading group can be imported as submesh. Maya needs to export with "Write Face Sets" option enabled. Please note that this option is off by default.
+Maya's shading group can be imported as a submesh. Maya needs to export with "Write Face Sets" option enabled. Please note that this option is off by default.
```

Comment:
I think we should not take any chance and add the (r) everytime we mention Maya. Can you search for Maya in the whole file and make the change please?

Reply:
It will look silly, I know... Maybe Amy can find other guidelines, but let's cover the angles ourselves and use what Autodesk recommends